src/patternfly/components/Label/label-editable-text.hbs

```diff
@@ -1,4 +1,4 @@
-<{{#if label-editable-text--type}}{{label-editable-text--type}}{{else if label--IsEditableActive}}input{{else}}button{{/if}} class="pf-c-label__editable-text{{#if label-editable-text--modifier}} {{label-editable-text--modifier}}{{/if}}"
+<{{#if label-editable-text--type}}{{label-editable-text--type}}{{else if label--IsEditableActive}}input{{else}}button{{/if}} class="pf-c-label__content pf-c-label__editable-text{{#if label-editable-text--modifier}} {{label-editable-text--modifier}}{{/if}}"
```

Comment:
Typically we don't share BEM elements like this `pf-c-label__content pf-c-label__editable-text`.

Is it necessary for this to have the `__editable-text` class, or could we just target it (including in react) by the `__content` class when the parent is `.pf-m-editable`?

Also I wonder if this logic should be built in to the `__content` conditional? wdyt?